scripts/audio.js

```diff
@@ -0,0 +1,78 @@
+// Create the new Tone.js instance
+const synth = new Tone.Synth().toMaster();
+
+// This is a dictionary object to store the songs we want to play
+let playlist = {
+  missionImpossible: [
+    "G2",
+    [null, "G2"],
+    null,
+    "Bb2",
+    "C3",
+    "G2",
+    [null, "G2"],
+    null,
+    "F2",
+    "F#2",
+  ],
+  twinkleTwinkle: [
+    "C2",
+    "C2",
+    "G2",
+    "G2",
+    "A2",
+    "A2",
+    "G2",
+    null,
+    "F2",
+    "F2",
+    "E2",
+    "E2",
+    "D2",
+    "D2",
+    "C2",
+    null,
+  ],
+};
```

Comment:
I'm a big fan personally of seperating data into seperate files. Its great for organization. Then you import or require the data in the main file. It makes the files easier to read and makes it easier if the project ever gets bigger, more complex.